test/FunFlow/TestFlows.hs

```diff
@@ -26,13 +24,22 @@ data FlowAssertion where
                 -> IO () -- test setup action
                 -> FlowAssertion
 
+flow2 :: Flow SomeException () (Double,Double)
+flow2 = proc () -> do
+  r1 <- worstBernoulli error -< 0.2
+  r2 <- worstBernoulli error -< 0.3
+  returnA -< (r1,r2)
+
+flow2caught :: Flow SomeException () (Double,Double)
+flow2caught = retry 100 0 flow2
+
 flowAssertions :: [FlowAssertion]
 flowAssertions =
   [ FlowAssertion "death" "foo" melancholicLazarus Nothing setup
-  , FlowAssertion "resurrection" "bar" (retry 1 1 melancholicLazarus) (Just "bar") setup
-  , FlowAssertion "bernoulli" 0.2 (retry 20 0 $ worstBernoulli (toException . toDyn) >>> arr (<2.0)) (Just True) (return ())
+  , FlowAssertion "resurrection" "bar" (retry 1 0 melancholicLazarus) (Just "bar") setup
+  , FlowAssertion "bernoulli_once" 0.2 (retry 20 0 $ worstBernoulli error >>^ (<2.0)) (Just True) (return ())
```

Comment:
`userError` would be better than straight `error` here, since it's at least the right type. I know `error` can be any type, but as such, its use here is somewhat confusing.

Reply:
thanks, i was looking for that last night